### PR TITLE
chore: Action create option order: Maintain parity with EE repo

### PR DESCRIPTION
**Pull request by hetunandu**
## Description
We had a few test cases failing on the EE repo because of different permissions and had to make some changes on that repo to get it pass the CI. This PR is copying the changes back to this repo for posterity

https://github.com/appsmithorg/appsmith-ee/pull/1421

### CODE REVIEW

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/OtherUIFeatures/Omnibar_spec.js [hunk 1]
```diff
@@ -81,35 +81,24 @@ describe("Omnibar functionality test cases", () => {
     cy.get(omnibar.categoryTitle).eq(1).click();
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/OtherUIFeatures/Omnibar_spec.js:81]: This index?

**Review by Aishwarya-U-R — COMMENTED** [review]
Approving Cypress changes with open questions due to urgency with fixing EE conflicts.

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/OtherUIFeatures/Omnibar_spec.js [hunk 2]
```diff
@@ -81,35 +81,24 @@ describe("Omnibar functionality test cases", () => {
...
-
-    cy.get(omnibar.createNew).eq(0).click();
+    cy.get(omnibar.createNew).contains("New JS Object").click();
     cy.wait(1000);
     cy.wait("@createNewJSCollection");
     cy.wait(1000);
     cy.get(".t--js-action-name-edit-field").type(jsObjectName).wait(1000);
+
     cy.get(omnibar.globalSearch).click({ force: true });
     cy.get(omnibar.categoryTitle).eq(1).click();
```
[the remark this answers is not in the document]
Comment (hetunandu) [reply app/client/cypress/integration/Regression_TestSuite/ClientSideTests/OtherUIFeatures/Omnibar_spec.js:91]: No, this was not changed in implementation. We can improve this test code later on to remove other indexes. For now, this element is always in index 1 no mater what

[the remark this answers is not in the document]
Comment (Aishwarya-U-R) [reply app/client/cypress/integration/Regression_TestSuite/ClientSideTests/OtherUIFeatures/Omnibar_spec.js:91]: Got it.

**Pull request merged**